projects/storefrontapp-e2e-cypress/cypress/sample-data/checkout-flow.ts

```diff
@@ -26,6 +26,32 @@ export interface SampleUser {
   };
 }
 
+export interface SimpleUser {
```

Comment:
What is this for? 

Can't we re-use SampleUser and make all fields optional

Reply:
We could as it's the same. I didn't know if there was a reason for these fields to not be optional.